iOS/Flows/Conversation/_New/ConversationHeaderView.swift

```diff
@@ -49,32 +48,18 @@ class ConversationHeaderView: View {
         self.button.set(style: .icon(image: UIImage(systemName: "plus")!, color: .background4))
     }
 
-    func configure(with item: Conversation) {
-        self.currentItem = item
+    func configure(with conversation: Conversation) {
+        self.label.setText(conversation.title)
+        self.descriptionLabel.setText(conversation.description)
 
-        Task {
-            await self.display(conversation: item)
+        let members = conversation.lastActiveMembers.filter { member in
+            return member.id != ChatClient.shared.currentUserId
         }
-    }
-
-    private func display(conversation: ChatChannel) async {
-        Task.onMainActor {
-            guard self.currentItem?.cid == conversation.cid else { return }
-
-            self.label.setText(conversation.title)
-            self.descriptionLabel.setText(conversation.description)
-
-            let members = conversation.lastActiveMembers.filter { member in
-                return member.id != ChatClient.shared.currentUserId
-            }
-
-            if !members.isEmpty {
-                self.stackedAvatarView.set(items: members)
-            } else {
-                self.stackedAvatarView.set(items: [User.current()!])
-            }
 
```

Comment:
No need to split this configuration code into two functions as far as I can tell.